Docs/user_guide/getting_started.md

```diff
@@ -96,6 +96,15 @@ To run Slicer-4.11-2020-09-30 on older debian (e.g. debian 9) you may also need:
 #### ArchLinux
 ArchLinux runs the `strip` utility by default; this needs to be disabled in order to run Slicer binaries.  For more information see [this thread on the Slicer Forum](https://discourse.slicer.org/t/could-not-load-dicom-data/14211/5).
 
+#### Fedora
+Install the dependencies:
+
+    sudo dnf install mesa-libGLU libnsl
+
+The included libcrypto.so.1.1 in Slicer-4.11 is incompatible with the system libraries used by Fedora. The fix, until it is updated, is to move/remove the included libcrypto files:
```

Comment:
Since the the documentation here is for "latest" and that is Slicer 4.13, then Slicer-4.11 info shouldn't be here. The specific version should probably be left out of here as the documentation is version controlled so it is known this is matched to Slicer 4.13. If at some point things change and it isn't incompatible, that should be updated in the user manual.  If this is an issue only known for Slicer Stable 4.11.20210226, then the documentation for the Slicer `v4.11` branch should be updated instead, not the `master` branch.

Reply:
Right, thanks for letting me know. I could only test Slicer 4.11 since the 4.13 download did not include libITKniftiio-5.3.so.1 and couldn't launch. I'll make an update in the 4.11 branch.

Reply:
Following this logic of "latest" being Slicer 4.13 the information under Debian/Ubuntu regarding "Slicer-4.11-2020-09-30 on older debian" should also be removed. Should I create another pull request for that?

Reply:
Yes that should be removed as well since the documentation should now be for the current version here in the `master` branch. You can push another commit to your branch here instead of another PR.

Reply:
> the 4.13 download did not include libITKniftiio-5.3.so.1 and couldn't launch

@jcfr do we have a Linux packaging error following recent commit https://github.com/Slicer/Slicer/commit/b1c235939e1adb34d22da839a51c838c4ff0a47e?

Reply:
Ok I'll make the updates here, after I can test with 4.13 to see if the libcrypto information is needed.

Reply:
@numas please try the latest Slicer Preview now which should have the Linux packaging error resolved. Thanks!

Reply:
Great, thank you! Unfortunately the included libcrypto issue is still there, so I've changed the text to try and cover both releases.

I may have been a bit trigger happy when creating the pull request for the 4.11 branch. Is it still relevant?

Reply:
Yes we'll let our developer more familiar with the Linux packaging take a look at it. Thanks!